pyproject.toml

```
@@ -9,7 +9,7 @@ include = ["src/*"]
 #set to false.  poetry-dynamic-versioning only needed for CI versioning.
 enable = false
 vcs = "git"
-style = "pep440"
+style = "semver"
```

This repo actually uses PEP-440 conforming tags. (ex: v2021.7)
```suggestion
style = "pep440"
```

---

Done.